snowfakery/tools/snowbench.py

```diff
@@ -40,9 +47,21 @@ def snowbench(
     execution process is spawned off into a sub-process instead
     of being executed inline.
     """
-    with TemporaryDirectory() as tempdir:
+    with TemporaryDirectory() as tempdir, click.progressbar(
+        label="Benchmarking", length=num_records
```

Comment:
```suggestion
        label="Benchmarking", length=num_records, show_eta=False
```

The ETA is off by several orders of magnitude, so I suggest we hide it.